### ISSUE TITLE
Accumulated distortion of sound while looping short video

### ISSUE THREAD

**Issue by Gigas002** [mpv-player/mpv#9472]
### Important Information

mpv version:

```text
mpv 0.34.0-58-gb89d3f205b Copyright © 2000-2021 mpv/MPlayer/mplayer2 projects
 built on Thu Nov 18 03:20:06 2021
FFmpeg library versions:
   libavutil       56.70.100
   libavcodec      58.134.100
   libavformat     58.76.100
   libswscale      5.9.100
   libavfilter     7.110.100
   libswresample   3.9.100
FFmpeg version: n4.4.1
```

linux destribution and version:

```text
Operating System: Manjaro Linux
Kernel: Linux 5.15.2-2-MANJARO
Architecture: x86-64
```

source of mpv binary: mpv-git from AUR: https://aur.archlinux.org/packages/mpv-git/

window manager: `x11, KDE Plasma 5.23.3`

GPU driver: `proprietary nvidia driver 495.44`

video: https://0x0.st/-R6R.mp4

additional info, `pactl info` output:

```text
Library Protocol Version: 35
Server Protocol Version: 35
Server Name: PulseAudio (on PipeWire 0.3.40)
Server Version: 15.0.0
```

### Reproduction steps

Run the attached video with the following command line arguments for some (~30s) amount of time and listen to sound distortion:

`mpv --no-config --log-file=output.txt --loop tachibana\ marika.mp4`

### Expected behavior

Video plays looped without any sound issues.

### Actual behavior

Accumulated sound distortion happens on each loop more and more.

### Log file

Log file: https://pastebin.com/raw/1EnGqRA0

### Sample files

[tachibana marika.tar.gz](https://github.com/mpv-player/mpv/files/7565449/tachibana.marika.tar.gz)

**Comment by tmm1** [comment]
I suspect this started after https://github.com/mpv-player/mpv/commit/b74c09efbf7c6969fc053265f72cc0501b840ce1. Can you confirm with an older release?

**Comment by Gigas002** [comment]
Unfortunately, I don't have older binaries to run. I tried building [v0.32.0 release branch](https://github.com/mpv-player/mpv/tree/release/0.32) with waf, but that wasn't successful on both my manjaro pc and ubuntu VM for different reasons.
I also tried to build it with [mpv-build](https://github.com/mpv-player/mpv-build) tool specifying `./use-mpv-custom v0.32.0`; tried to `makepkg` from `mpv-full` [aur's 0.32.0 version](https://aur.archlinux.org/cgit/aur.git/commit/?h=mpv-full&id=20a17407990a518f43a2023857da73991198b401) but all failed. Sorry.

**Comment by Gigas002** [comment]
Tested with the latest commits, the problem seems to be resolved

**Comment by Dudemanguy** [comment]
Presumably fixed by #9480.
